core/src/test/scala/org/elasticmq/actor/QueueActorMsgOpsTest.scala

```diff
@@ -147,7 +147,7 @@ class QueueActorMsgOpsTest extends ActorTest with QueueManagerForEachTest with D
     val q1 = createQueueData("q1", MillisVisibilityTimeout(1L))
 
     for {
-      Right(queueActor) <- queueManagerActor ? CreateQueue(q1)
+      Right(queueActor) <- queueManagerActor ? CreateQueue(q1.copy(maxReceiveCount = 2))
```

Comment:
seems there is inconsistent in sqs docs:
Here it says that maxReceiveCount is not required - http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-sqs-queues-redrivepolicy.html
While here it should be set from 1 to 1000: http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-sqs-queues-redrivepolicy.html